images/openstack-venv-builder/Dockerfile

```diff
@@ -19,6 +19,7 @@ ADD https://releases.openstack.org/constraints/upper/master /upper-constraints.t
 RUN <<EOF sh -xe
 sed -i '/glance-store/d' /upper-constraints.txt
 sed -i '/horizon/d' /upper-constraints.txt
+sed -i '/python-openstackclient/d' /upper-constraints.txt
```

Comment:
I think this is dangerous here because then it will fully uncap `python-openstackclient` which will break other things, it's probably best we just don't install from source for clients (for now).

Reply:
@mnaser the only way I was able to get it to work is to source related aliases:

```
  tasks:
    - name: Test
      changed_when: false
      ansible.builtin.shell: |
        source /etc/profile.d/atmosphere.sh 
        openstack port list
      environment:
        OS_CLOUD: atmosphere
```
The newest (2.2.0) openstack.cloud collection like openstacksdk >= 1.0.0 also lacks this feature.

Reply:
@fitbeard Let's source it for now, it's not the prettiest but that whole thing is a hack, and then ideally we can get rid of the shell/command calls later.